src/Handler.ts

```diff
@@ -84,14 +82,17 @@ export class Handler {
     // make the tooltip visible
     this.el.classList.add('visible', `${this.options.theme}-theme`);
 
-    const {x, y} = calculatePosition(
-      event,
-      this.el.getBoundingClientRect(),
-      this.options.offsetX,
-      this.options.offsetY,
-    );
-
-    this.el.style.top = `${y}px`;
-    this.el.style.left = `${x}px`;
+    const position =
+      this.options.anchor === 'mark'
+        ? calculatePositionRelativeToMark(handler, event, item, this.el.getBoundingClientRect(), this.options)
+        : calculatePositionRelativeToCursor(
+            event,
+            this.el.getBoundingClientRect(),
+            this.options.offsetX,
+            this.options.offsetY,
```

Comment:
Let's refactor this to just take `this.options` so it's the same as above and can be combined as I suggested earlier.

Reply:
Yes, the would mean passing maybe unused arguments but it simplifies the call here.

Reply:
Yeah I like this suggestion better as well.

Reply:
Sorry I missed that in your ternary suggestion.